module-system/module-implementations/sov-ibc/src/context/clients.rs

```diff
@@ -314,14 +314,28 @@ where
         client_id: &ClientId,
         height: &ibc::Height,
     ) -> Result<Option<Self::AnyConsensusState>, ContextError> {
-        todo!()
+        let client_cons_state_path = ClientConsensusStatePath::new(client_id, &height.increment());
```

Comment:
The "next" consensus state is not necessarily at the next height (and actually most of the time it isn't). It could have come an arbitrary number of heights in the future. Refer to the [basecoin implementation](https://github.com/informalsystems/basecoin-rs/blob/aa59797863272119a7bcb63bdf13b6d037dcd5cf/crates/app/src/modules/ibc/client_contexts.rs#L85-L117).